src/vmm_mad/exec/one_vmm_exec.rb

```diff
@@ -529,7 +538,7 @@ def restore(id, drv_message)
     # MIGRATE (live) action, migrates a VM to another host creating network
     #
     def migrate(id, drv_message)
-        action = VmmAction.new(self, id, :migrate, drv_message)
+        action = VmmAction.new(self, id, :migrate, drv_message, true)
```

Comment:
The `true` can't be hard coded here, it must follow the `delegate_actions`.
And, similar change must be for all other actions.